subprojects/docs/src/docs/userguide/getting_started.adoc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[getting_started]]
+= Getting Started
+
+Everyone has to start somewhere and if you're new to Gradle, then this is a good place to begin.
+
+== What is Gradle?
+
+Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool that is designed to be flexible enough to build almost any type of software. It:
+
+ * _Runs on the JVM_ — but it can build anything, including native libraries and executables
+ * _Runs fast_ — particularly with well-designed builds
+ * _Uses conventions and plugins_ — which make it simple to build common types of project
+ * _Is customizable and extensible_
+ * _Can be run from the command line or IDEs_ — like Android Studio, IntelliJ IDEA and Eclipse
+ * _Grants insight into your builds_ — https://scans.gradle.com/[build scans] provide extensive information about a build run that you can also share
+
+As with many flexible and customizable tools, Gradle can take a little time to understand. However, you can get a jump start by reading this short primer:
+
+ * <<five_things#five_things,5 things you need to know about Gradle>>
+
+== Short tutorials
```

Comment:
It feels like we ought to have a brief "Installing" section similar to this: https://docs.bazel.build/versions/master/getting-started.html#installation

Reply:
Why have that in addition to the high-level _Installing Gradle_ page? Which is just below the _Getting Started_ one. I don't feel that strongly about it, but it seems a little overkill.

I can understand in one sense, because installation is typically part of getting started. OK, I'll probably include it.

Anyway, I think I need to give some thought and attention to this page. Most of my energy was focused on the Five Things and Groovy primer pages, although I should probably take another look at the latter too.

Reply:
Great. Thanks for working through this. Overall we will produce something far superior to what we have today.